app/services/media-gallery/media-gallery.ts

```diff
@@ -28,7 +28,8 @@ export interface IMediaGalleryInfo extends IMediaGalleryLimits {
 }
 
 interface IMediaGalleryProps {
-  audioOnly: boolean;
+  audioOnly?: boolean;
+  imageOnly?: boolean;
```

Comment:
You can't set both, right? Maybe use something like `filter: {images: true, audio: true }` ?

Reply:
yea that's true, i'll look into making a better pattern here